app/component/SelectedStopPopupContent.js

```diff
@@ -1,10 +1,20 @@
 import PropTypes from 'prop-types';
 import React from 'react';
+import { FormattedMessage } from 'react-intl';
 
 const SelectedStopPopupContent = ({ stop }) => (
   <div className="origin-popup">
     <div className="origin-popup-header">
-      <div className="selected-stop-header">{stop.name}</div>
+      <div className="selected-stop-header">
+        {!stop.name || stop.networks[0] === 'tier_REUTLINGEN' ? (
+          <FormattedMessage
```

Comment:
We should not use network specific code here. Is there a way to deduce the vehicleType from the stop and use a vehicleType specific message?

Reply:
I used this solution because originally the popup should show the `name` of the actual stop - but in case of this network, the `name` is an incomprehensible number code. If we'd use any other condition, this process will change. Should we refactor this anyway and do not use the `name` (not at any stop popup) but display a message based on the vehicle type?